fastlane/lanes/release.rb

```diff
@@ -101,7 +98,7 @@
     )
 
     unless skip_user_confirmation || UI.confirm('Ready to push changes to remote to let the automation configure it on GitHub?')
-      UI.message('Aborting code freeze as requested.')
+      UI.message("Terminating as requested. Don't forget to run the remainder of this automation manually.")
```

Comment:
From https://github.com/wordpress-mobile/WordPress-iOS/pull/22191#discussion_r1424460296

> but this message feels a bit off to me. When we run this lane locally, the code freeze is completed at this point - we just didn't push it to remote yet.
>
> It does make sense in CI, because if the change is not pushed, then it's discarded. However, that's not the case for local. 🤷

Changed the verb to "terminating" which I hope removes the impression that the lane is atomic and therefore aborting would discard changes. Also added the second sentence to reinforce the fact that changes have been made and it's up to the user to complete the process.

I considered listing what the remaining steps were, but then decided against it to avoid having to maintain that list as the lane evolves.

Reply:
I think this works OK as a generic message. Thanks for the improvement!